### PR TITLE
Cotco 63

### PR DESCRIPTION
**Pull request by andreicostin**
HTML with variables + styling

### CODE REVIEW

File: app/uk/gov/hmrc/ctc/config/AppConfig.scala [hunk 1]
```diff
@@ -32,9 +33,14 @@ class AppConfig @Inject()(val runModeConfiguration: Configuration, environment:
 
   lazy val ctcBaseUrl = s"${baseUrl("common-transit-convention")}/common-transit-convention"
 
-  lazy val assetsPrefix = loadConfig(s"assets.url") + loadConfig(s"assets.version")
-  lazy val analyticsToken = loadConfig(s"google-analytics.token")
-  lazy val analyticsHost = loadConfig(s"google-analytics.host")
+  lazy val ports: List[String] = {
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/config/AppConfig.scala]: Nice change. Still, would you please give a more informative name than just **ports** ? Reading just as **ports** I was thinking it was something related to the network. Something like portsOfEntry or destinationPorts or ...

**Comment by robertbutacu** [comment]
Do not close the PR as I would like to update the Model for the Data and also have a feature toggle which either connects or not to the backend

File: app/uk/gov/hmrc/ctc/controllers/TransitController.scala [hunk 2]
```diff
@@ -18,22 +18,44 @@ package uk.gov.hmrc.ctc.controllers
...
   (implicit appConfig: AppConfig) extends FrontendController with I18nSupport {
 
-  def showTransitPage(destinationPort: String, transitState: String) = Action.async { implicit request =>
+  def showTransitPage(destinationPort: String, transitState: String): Action[AnyContent] = Action.async { implicit request =>
     ctcService.getNextTransitByPoEAndState(destinationPort, transitState).map(transit =>
         Ok(views.html.transits(destinationPort, transitState, transit))
     )
   }
 
+  def showStatsPage(): Action[AnyContent] = Action.async {
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/controllers/TransitController.scala]: I will consider this just as a quick trick ;)

File: app/uk/gov/hmrc/ctc/views/peggy_screen.scala.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
+@*
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/views/peggy_screen.scala.html]: Would you please rename the template to `ManagerView.scala.html` ?

Comment (udinobi) [reply app/uk/gov/hmrc/ctc/views/peggy_screen.scala.html]: ... or `manager_view.scala.html` ?

File: app/uk/gov/hmrc/ctc/config/AppConfig.scala (hunk 1 above)
Comment (andreicostin) [reply app/uk/gov/hmrc/ctc/config/AppConfig.scala]: @robertbutacu

**Comment by andreicostin** [comment]
@udinobi can you check again please

**Pull request merged**